Cargo.toml

```diff
@@ -74,12 +79,13 @@ pretty-bytes = "0.2.2"
 
 [features]
 blt-backend = ["bear-lib-terminal"]
-default = ["ncurses-backend"]
+default = ["crossterm"]
 markdown = ["pulldown-cmark"]
 ncurses-backend = ["ncurses", "maplit", "term_size"]
 pancurses-backend = ["pancurses", "maplit", "term_size"]
 termion-backend = ["termion"]
+crossterm-backend = ["crossterm"]
 unstable_scroll = []
 
 [lib]
-name = "cursive"
+name = "cursive"
```

Comment:
Somehow it seems your editor is eating away the trailing newlines in this file (and in the `.gitignore`).